### PR TITLE
New Resource `alicloud_ga_accelerator_spare_ip_attachment`; New Datasource `alicloud_ga_accelerator_spare_ip_attachments`;

### PR DESCRIPTION
**Pull request by super-eggs**
New Resource `alicloud_ga_accelerator_spare_ip_attachment`;
New Datasource `alicloud_ga_accelerator_spare_ip_attachments`;

### CODE REVIEW

File: alicloud/data_source_alicloud_ga_accelerator_spare_ip_attachments.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		Schema: map[string]*schema.Schema{
+			"accelerator_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"dry_run": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
```
Comment (xinmu0426) [inline alicloud/data_source_alicloud_ga_accelerator_spare_ip_attachments.go:27]: dry_run这个去掉

File: alicloud/resource_alicloud_ga_accelerator_spare_ip_attachment.go [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+		Schema: map[string]*schema.Schema{
+			"accelerator_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"dry_run": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_ga_accelerator_spare_ip_attachment.go:35]: dry_run 不需要ForceNew

File: alicloud/resource_alicloud_ga_accelerator_spare_ip_attachment_test.go [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    auto_use_coupon        = true
+}
+resource "alicloud_ga_bandwidth_package_attachment" "default" {
+	accelerator_id = alicloud_ga_accelerator.default.id
+	bandwidth_package_id = alicloud_ga_bandwidth_package.default.id
+}
+`, name)
+}
+
+func TestAccAlicloudGaAcceleratorSpareIpAttachment_unit(t *testing.T) {
```
Comment (super-eggs) [inline alicloud/resource_alicloud_ga_accelerator_spare_ip_attachment_test.go:100]: 单元测试覆盖率 95.3%

Comment (xiaozhu36) [reply alicloud/resource_alicloud_ga_accelerator_spare_ip_attachment_test.go:100]: 以后单元测试的名称要改成 TestUnitAlicloudxxxx，具体的可以问问倪洪磊

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**